console/frontend/pom.xml

```diff
@@ -107,13 +111,11 @@
 							<pluginExecutions>
 								<pluginExecution>
 									<pluginExecutionFilter>
-										<groupId>com.github.eirslett</groupId>
-										<artifactId>frontend-maven-plugin</artifactId>
-										<versionRange>[1.12,)</versionRange>
+										<groupId>org.codehaus.mojo</groupId>
+										<artifactId>exec-maven-plugin</artifactId>
+										<version>3.3.0</version>
 										<goals>
-											<goal>install-node-and-npm</goal>
-											<goal>npm</goal>
-											<goal>npx</goal>
+											<goal>exec</goal>
```

Comment:
Dit mag ook weg.

Reply:
Ah oke, voor de eirslett plugin was van alles opgezet en ik wist niet of dit een eclipse dingentje was dus had het maar geprobeerd na te maken